BEIMA.Backend.FT/BuildingFT.cs

```diff
@@ -66,6 +79,25 @@ await TestClient.GetBuilding(id)
             Assert.That(ex?.StatusCode, Is.EqualTo(HttpStatusCode.NotFound));
         }
 
+        public void NoBuildingsInDatabase_Unauthorized_BuildingGet_ReturnsUnauthorized(string id)
+        {
+            var ex = Assert.ThrowsAsync<BeimaException>(async () =>
+                await UnauthorizedTestClient.GetBuilding("1234567890abcdef12345678")
+            );
+            Assert.IsNotNull(ex);
+            Assert.That(ex?.Message, Does.Contain("Invalid credentials."));
+            Assert.That(ex?.StatusCode, Is.EqualTo(HttpStatusCode.Unauthorized));
+        }
+
+        public void NoBuildingsInDatabase_Unauthorized_BuildingGet_EmptyId_ReturnsNotFound()
```

Comment:
We already have a test case for this on NoBuildingsInDatabase_BuildingGet_ReturnsNotFound

Reply:
That test case doesn't test if the client is unauthorized though.

Reply:
That's true, but this one still seems to be a not found error over an unauthorized error. But I suppose it shouldn't hurt to leave it in.